src/pages/base/manage-casetypecategory-list.page.ts

```diff
@@ -43,9 +72,18 @@ export abstract class ManageCasetypecategoryList extends BasePage {
   }
 
   /**
+   * @param {Page} filters filter object
    * @returns page title
    */
-  getPageUrl (): string {
-    return `/civicrm/case/a/?case_type_category=${this.getCaseTypeCategoryValue()}#/case/list?cf=%7B%22case_type_category%22:%22${this.getCaseTypeCategoryValue()}%22%7D`;
+  getPageUrl (filters?: any): string {
+    let baseUrl = `/civicrm/case/a/?case_type_category=${this.getCaseTypeCategoryValue()}#/case/list?cf=%7B%22case_type_category%22:%22${this.getCaseTypeCategoryValue()}%22`;
+
+    if (filters?.status !== undefined) {
+      baseUrl += `,%22status_id%22:%5B%22${this.getCaseStatusID(filters.status)}%22%5D`;
+    }
+
+    baseUrl += '%7D';
```

Comment:
I think we should try to use "human-readable symbols" here return the ones expected by the browser using `encodeURI`

This way, we can also JSON encode the filters instead of picking them individually:

```ts
getPageUrl(filters: Record<string, any> = {}) { // default value is {}
  const baseUrl = `....?cf=${JSON.stringify(filters}`;

  return encodeURI(baseUrl);
}
```

I'd resolve the filters in the test itself:

```ts
await manageCases.navigate(global.page, {
  status_id: await getCaseStatusIdByName('Resolved'),
});
```